modules/crud/sql/describe.go

```diff
@@ -41,9 +70,29 @@ func (s *SQL) getDescribeDetails(ctx context.Context, col string) ([]utils.Field
 	return result, nil
 }
 
-func (s *SQL) getForeignKeyDetails(ctx context.Context, project, col string) ([]utils.ForeignKeysType, error) {
-	fmt.Println("Sql", project, col)
-	rows, err := s.client.Queryx("select TABLE_NAME, COLUMN_NAME, CONSTRAINT_NAME, REFERENCED_TABLE_NAME, REFERENCED_COLUMN_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE REFERENCED_TABLE_SCHEMA = '" + project + "' and TABLE_NAME = '" + col + "'")
+func (s *SQL) getForeignKeyDetails(ctx context.Context, project, dbType, col string) ([]utils.ForeignKeysType, error) {
+	queryString := ""
+	if utils.DBType(dbType) == utils.MySQL {
+		queryString = "select TABLE_NAME, COLUMN_NAME, CONSTRAINT_NAME, REFERENCED_TABLE_NAME, REFERENCED_COLUMN_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE REFERENCED_TABLE_SCHEMA = :project and TABLE_NAME = :col"
+	} else {
+		queryString = `SELECT
```

Comment:
Which database is this going to search the table in

Reply:
same here